3.3-nginx/nginx.conf

```diff
@@ -3,27 +3,16 @@ load_module modules/ngx_http_modsecurity_module.so;
 user ${USER};
 worker_processes 1;
 
-error_log ${ERRORLOG} ${LOGLEVEL};
 pid /var/run/nginx.pid;
 
 events {
     worker_connections ${WORKER_CONNECTIONS};
 }
 
 http {
-
-    modsecurity on;
-    modsecurity_rules_file /etc/modsecurity.d/setup.conf;
-
     include /etc/nginx/mime.types;
     default_type application/octet-stream;
 
-    log_format main '$remote_addr - $remote_user [$time_local] "$request" '
-                    '$status $body_bytes_sent "$http_referer" '
-                    '"$http_user_agent" "$http_x_forwarded_for"';
-
-    access_log /var/log/nginx/access.log main;

```

Comment:
Explanation: This value is available from [coreruleset/modsecurity-docker:/v3-nginx/conf.d/logging.conf](https://github.com/coreruleset/modsecurity-docker/blob/master/v3-nginx/conf.d/logging.conf#L5)